client/src/app/shared/services/ingest.service.ts

```diff
@@ -81,10 +81,23 @@ export class IngestService {
       .pipe(map(data => new Account({
```

Comment:
This looks similar to what's happening in line 92. I think it merits a refactor.